src/utils/dom-tranform-stream.js

```diff
@@ -147,9 +153,19 @@ export class DomTransformStream {
         this.currentChunkTransferPromise_ = null;
         const targetBody = resolvedElements[0];
         removeNoScriptElements(dev().assertElement(this.detachedBody_));
-        while (this.detachedBody_.firstChild) {
+
+        // Always leave one node that may get updated on next chunk.
+        while (this.detachedBody_.children.length > 1) {
```

Comment:
`children` !== `firstChild`. The `children` API operates with elements, while the `firstChild` and fam operate with nodes. E.g. try to test with a text-node child and you will see the delta. I think `element.childNodes` would work for you here.

Reply:
Changed to `childNodes`.